staging/src/k8s.io/mount-utils/mount_helper_unix.go

```diff
@@ -61,7 +61,7 @@ func IsCorruptedMnt(err error) bool {
 		underlyingError = err
 	}
 
-	return underlyingError == syscall.ENOTCONN || underlyingError == syscall.ESTALE || underlyingError == syscall.EIO || underlyingError == syscall.EACCES || underlyingError == syscall.EHOSTDOWN || underlyingError == syscall.EWOULDBLOCK
+	return underlyingError == syscall.ENOTCONN || underlyingError == syscall.ESTALE || underlyingError == syscall.EIO || underlyingError == syscall.EACCES || underlyingError == syscall.EHOSTDOWN || underlyingError == syscall.EWOULDBLOCK || underlyingError == syscall.ENODEV
```

Comment:
there is golint error, could you use `errors.Is(underlyingError, syscall.ENODEV)` ?

Reply:
Ack, fixed in https://github.com/kubernetes/kubernetes/pull/126174/commits/14b8f2ae5256f08913ce667605b4e62717c5ae55